connector_test.go

```diff
@@ -450,28 +416,28 @@ func TestNetworkHost(t *testing.T) {
 	// The first container will run with the host namespace
 	configtemplate := fmt.Sprintf(networkTemplate, containerNamePrefix, "host")
 	connector, _ := getConnector(t, configtemplate)
-	plugin, err := connector.Deploy(context.Background(), "quay.io/tsebastiani/arcaflow-engine-deployer-podman-test:latest")
+	plugin, err := connector.Deploy(
+		context.Background(),
+		"quay.io/tsebastiani/arcaflow-engine-deployer-podman-test:latest")
 	assert.NoError(t, err)
 
 	var containerInput = []byte("network host\n")
 	// the test script will run "ifconfig" in the container
 	assert.NoErrorR[int](t)(plugin.Write(containerInput))
+
 	var ifconfigOut bytes.Buffer
 	// runs ifconfig in the host machine in order to check if the container has exactly the same network configuration
 	cmd := exec.Command("/bin/bash", "-c", "ifconfig | grep -P \"^.+:\\s+.+$\" | awk '{ gsub(\":\",\"\");print $1 }'")
 	cmd.Stdout = &ifconfigOut
-	err = cmd.Run()
-	assert.Nil(t, err)
+	assert.NoError(t, cmd.Run())
+
 	ifconfigOutStr := ifconfigOut.String()
 	logger.Infof(ifconfigOutStr)
 	readBuffer := readOutputUntil(t, plugin, ifconfigOutStr)
-	if len(readBuffer) == 0 {
-		t.Fatal("the container did not produce any output")
-	}
+	assert.Equals(t, len(readBuffer) > 0, true)
+
 	containerOutString := string(readBuffer)
-	if strings.Contains(containerOutString, ifconfigOutStr) == false {
-		t.Fatalf("expected string not found: %s", ifconfigOutStr)
-	}
+	assert.Equals(t, strings.Contains(containerOutString, ifconfigOutStr), true)
```

Comment:
This should be switched to the contains assertion to preserve the context: https://github.com/arcalot/go-assert/blob/9911908433a59f9fecfbdc9cd872c8e26d0ebad9/equality.go#L10